### PR TITLE
Navigate to application pages

### LINKED ISSUE #2: 2. As a user, I want to be able to navigate to all the pages in the application by clicking the links in the nav bar
## Summary

Users need to be able to navigate between the pages of our application in order to access all of its features. There is a navigation bar in the `Layout.jsx` component, but it's not quite feature-complete: the links in the nav element don't actually allow the user to navigate from page to page.

## Acceptance criteria
- [ ] The `nav` element is added to the `Layout` component
- [ ] Links to the "Home", "List", and "Add item" pages of the app are In the `nav` element
- [ ] The links all function as expected using  the `NavLink` component from `react-router-dom`

## Notes
- The [official docs for the `NavLink` component](https://reactrouterdotcom.fly.dev/docs/en/v6/components/nav-link) will come in very handy!
- While a standard anchor element (`<a>`) _could_ allow a user to navigate from page to page, the `NavLink` component adds additional behavior that is essential to our app functioning correctly!

### PR DESCRIPTION
**Pull request by thatgirldorian**
## Description

This PR adds functionality to the navigation links in the shopping list application using the NavLink component. Each link goes to the respective page: Home, List and Add item.

## Related Issue
Closes #2 

## Acceptance Criteria
- [x] The `nav` element is added to the `Layout` component
- [x] Links to the "Home", "List", and "Add item" pages of the app are In the `nav` element
- [x] The links all function as expected using  the `NavLink` component from `react-router-dom`

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

## Testing Steps / QA Criteria
* From your terminal, pull down this branch with `git pull origin do-ve-navigate-to-all-pages` and check that branch out with `git checkout do-ve-navigate-to-all-pages` 
* Then `npm ci` to install the newly added dependencies locally and `npm start` to launch the app.
* Click on all three navigation links to move to new pages

### CODE REVIEW

**Comment by Terieyenike** [comment]
I think you should include the import component from `react-router-dom` in one line instead of two different imports just like this `import { Outlet, NavLink } from 'react-router-dom';` 

Also, remove all the comments from the completed code.

Good job, guys, @thatgirldorian @mohanner-coder

**Comment by thatgirldorian** [comment]
Thanks @Terieyenike ! :)

I just made these changes.

File: CONTRIBUTORS.md
Comment (pmbanugo) [inline CONTRIBUTORS.md]:
```suggestion
```

File: CONTRIBUTORS.md
Comment (pmbanugo) [inline CONTRIBUTORS.md]:
```suggestion
```

**Review by pmbanugo — COMMENTED** [review]
Well done 👏🏽

File: CONTRIBUTORS.md
Comment (pmbanugo) [inline CONTRIBUTORS.md]: These links are the same as the ones below. Pls remove the duplicates

**Review by pmbanugo — COMMENTED** [review]
One final request from my part. You should get a review and approval from the other team before I approve mine.

Tip: The PR title should describe what the PR is about in a few words. With what you have at the moment, it is not clear what the PR should do so I'd recommend updating it.

File: CONTRIBUTORS.md
Comment (thatgirldorian) [reply CONTRIBUTORS.md]: I just cleaned up the duplicates and redid the PR title. Appreciate the feedback! :)

**Comment by thatgirldorian** [comment]
Hi, @faysvas I just worked on the changes based on Peter's feedback. Please let me know what you think! Thanks!

**Review by faysvas — APPROVED** [review]
Great job!

**Pull request merged**
